eos/form-factors/baryonic-processes.hh

```diff
@@ -55,6 +55,27 @@ namespace eos
         static constexpr const std::tuple<QuarkFlavor, QuarkFlavor> partonic_transition = std::make_tuple(QuarkFlavor::bottom, QuarkFlavor::charm);
     };
 
+    struct LambdaCToLambda {
+        static constexpr const char * label = "Lambda_c->Lambda";
+        static constexpr const char * name_1 = "mass::Lambda_c";
+        static constexpr const char * name_2 = "mass::Lambda";
+        static constexpr const std::tuple<QuarkFlavor, QuarkFlavor> partonic_transition = std::make_tuple(QuarkFlavor::charm, QuarkFlavor::strange);
+        // initial state mass
+        static constexpr double m1 = 2.2865;
+        // final state mass
+        static constexpr double m2 = 1.115683;
+        // semileptonic kinematic endpoint
+        static constexpr double tm = (m1 - m2) * (m1 - m2);
+        // OPE results for the unitarity bounds
+        // TODO:
```

Comment:
Is there still something left to-do here? If so, the comment should be more specific. Or if not remove

Reply:
Yes. Pending.